src/components/available-organisation-units-tree/AvailableOrganisationUnitsTree.js

```diff
@@ -22,92 +23,69 @@ const defaultStyles = {
 }
 
 export class AvailableOrganisationUnitsTree extends PureComponent {
-    static propTypes = {
-        d2: PropTypes.object.isRequired,
-        onChange: PropTypes.func,
-        style: PropTypes.object,
-    }
-
-    static defaultProps = {
-        onChange: null,
-        style: defaultStyles,
-    }
-
-    constructor() {
-        super()
-
-        this.state = {
-            selected: [],
-            rootWithMember: null,
-        }
+    state = {
+        selected: [],
```

Comment:
Why not putting the `selected` state into redux or is there a reason for keeping it here?
If there's no more state, this component can be turned into a stateless functional component which will make it easier to add hooks if we ever need them in the future with react v17.